plugins/node/opentelemetry-instrumentation-runtime/src/instrumentation.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { InstrumentationBase } from "@opentelemetry/instrumentation";
+import { createEventLoopLagMetrics } from "./metrics/event-loop-lag";
+import { RuntimeInstrumentationConfig } from "./types";
+import { VERSION } from "./version";
+
+/**
+ * Runtime instrumentation for Opentelemetry
+ */
+export class RuntimeInstrumentation extends InstrumentationBase {
+  constructor(_config: RuntimeInstrumentationConfig = {}) {
+    super("@opentelemetry/instrumentation-runtime", VERSION, _config);
+  }
+
+  init() {
+    createEventLoopLagMetrics(this.meter, this._config);
+  }
+}
```

Comment:
I am not sure if this is the approach for "metric only" instrumentation. The `host-metrics` module is not using this approach[1]. 

[1] https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/packages/opentelemetry-host-metrics